driver/fixtures/fixtures_test.go

```diff
@@ -35,11 +36,11 @@ var Suite = &fixtures.Suite{
 			"NoopSameLine",
 			"Str",
 			"StringLiteral",
+			"alias",
 			"arg",
 			"kwarg",
 			"kwonly_arg",
 			"vararg",
-			"alias",
 		},
 	},
```

Comment:
If you like, you can also try to enable the token verification. Here is an example for [C#](https://github.com/bblfsh/csharp-driver/blob/master/driver/fixtures/fixtures_test.go#L46). Although Python's tokenizer may fail it. But maybe at least for some nodes it doesn't?